tests/test_pep518.py

```diff
@@ -15,10 +15,9 @@
 @pytest.mark.parametrize(
     'build_arg', ['', '--wheel'], ids=['sdist_to_wheel', 'wheel_directly']
 )
-@pytest.mark.xfail(sys.platform.startswith('cygwin'), reason='ninja pip package not available on cygwin', strict=True)
 def test_pep518(pep518, package, build_arg, tmp_path):
     dist = tmp_path / 'dist'
 
-    with cd_package(package) as package_dir, in_git_repo_context():
+    with cd_package(package), in_git_repo_context():
         build_args = [build_arg] if build_arg else []
-        subprocess.run([sys.executable, '-m', 'build', f'--outdir={dist}', *build_args], cwd=package_dir, check=True)
+        subprocess.check_call([sys.executable, '-m', 'build', '--outdir', str(dist), *build_args])
```

Comment:
Why are you replying `run` with `check_call`? `check_call` and `check_output` are the old, pre Python 3.5 API, and the Python docs recommend `run` instead. The rest of the calls should be changed to runs long-term, I think?

Reply:
Oh, sorry, I saw that the rest of the code uses the "old" API and I assumed it is the recommended one. Switching it back and probably adding a commit to clean up the other call sites too.